src/Framework/ResponseGeneratedFileUsedEventArgs.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Build.Framework
+{
+    /// <summary>
+    /// Arguments for the response file used event
+    /// </summary>
+    [Serializable]
+    public class ResponseGeneratedFileUsedEventArgs : BuildMessageEventArgs
```

Comment:
The `Response` prefix looks out of place here. I don't think that these files are considered "response". Unlike response files handled by `ResponseFileUsedEventArgs`, which maybe this class is modeled after?

```suggestion
    public class GeneratedFileUsedEventArgs : BuildMessageEventArgs
```

Reply:
Thanks! Done